src/Exceptionless.Web/Utility/Handlers/OverageMiddleware.cs

```diff
@@ -87,14 +87,15 @@ public sealed class OverageMiddleware {
 
         var organization = await organizationTask;
         var project = await projectTask;
-        bool overLimit = await _usageService.IncrementUsageAsync(organization, project, tooBig);
 
         // block large submissions, client should break them up or remove some of the data.
         if (tooBig) {
+            await _usageService.IncrementTooBigAsync(organization, project).AnyContext();
```

Comment:
Is this a new counter?

Reply:
It's a new utility method around increment.